build/windows/omsupply-build.bat

```diff
@@ -17,7 +17,12 @@ start /wait /b build\windows\omsupply-prepare.bat
 @if %errorlevel% neq 0 exit /b %errorlevel%
 
 @ECHO ##### Building omsupply for sqlite #####
-cd server && cargo build --release --bin omsupply_service && copy "target\release\omsupply_service.exe" "..\omSupply\Server\omSupply-sqlite.exe"
+@cd server 

```

Comment:
the rearranging isn't necessary for the android build, it just shifts the sqlite cargo compilation to be together, in the hope that it reduces some of the compilation time.